script.go

```diff
@@ -513,6 +514,10 @@ const (
 	// ScriptStrictMultiSig defines whether to verify the stack item
 	// used by CHECKMULTISIG is zero length.
 	ScriptStrictMultiSig
+
+	// ScriptDiscourageUpgradableNops defines whether to verify that
+	// NOP1 to NOP10 are reserved for future soft-fork upgrades.
+	ScriptDiscourageUpgradableNops
```

Comment:
Please also call out here that this flag must not be used for consensus critical code or applied to blocks and is only for enabling more strict standard transaction checks as well as the fact it only applies when the opcodes are actually executed.